FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/EditContactActivity.kt

```diff
@@ -34,21 +34,22 @@ import com.flowcrypt.email.util.GeneralUtil
  */
 class EditContactActivity : BaseImportKeyActivity(),
   UpdatePublicKeyOfContactDialogFragment.OnKeySelectedListener {
-  private val contactsViewModel: ContactsViewModel by viewModels()
-  private var contactEntity: ContactEntity? = null
+  private val recipientsViewModel: RecipientsViewModel by viewModels()
+  private var publicKeyEntity: PublicKeyEntity? = null
   private var editTextNewPubKey: EditText? = null
 
   override val contentViewResourceId: Int = R.layout.activity_edit_pgp_contact
   override val isPrivateKeyMode: Boolean = false
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    contactEntity = intent?.getParcelableExtra(KEY_EXTRA_CONTACT)
-    if (contactEntity == null) {
+    publicKeyEntity = intent?.getParcelableExtra(KEY_EXTRA_PUB_KEY)
+    if (publicKeyEntity == null) {
       finish()
```

Comment:
What does the `extra` mean here?

Reply:
Android uses `Bundle` to store some data and share it between different components(activities, services and etc.). To de short, `Bundle` it's like a map. So `KEY_EXTRA_PUB_KEY` it is just a key for some value.